beacon-chain/rpc/eth/beacon/lightclient.go

```diff
@@ -322,7 +343,7 @@ func (bs *Server) getLightClientEventBlock(ctx context.Context, minSignaturesReq
 
 	// Loop through the blocks until we find a block that has super majority of sync committee signatures (2/3)
 	var numOfSyncCommitteeSignatures uint64
-	if syncAggregate, err := block.Block().Body().SyncAggregate(); err == nil {
+	if syncAggregate, err := block.Block().Body().SyncAggregate(); err == nil && syncAggregate != nil {
```

Comment:
is this just general error checking or does syncAggregate != nil imply something about the majority?

Reply:
This is a general checking to avoid potential NPE in next call.